### ISSUE TITLE
isKAnonymized uses strings in stead of int

### ISSUE THREAD

**Issue by jacobInav** [uvm-plaid/programming-dp#20]
When you define isKAnonymized in chapter 3, the query assumes the numbers to be strings, not ints. This causes it to fail to detect anything in your examples.

**Comment by coding-famer** [comment]
yes I think it should be 
```python
query = ' & '.join([f'{col} == {row[col]}' for col in df.columns])
```
here

**Comment by jnear** [comment]
Fixed by 698169f91f844c44a858500a0a08b7ea6eb6301f

Thank you both!
